src/server/macos.rs

```diff
@@ -525,22 +564,26 @@ pub fn get_server_path(slot: &str) -> PathBuf {
         .join("bin/edgedb-server")
 }
 
-fn plist_dir(system: bool) -> anyhow::Result<PathBuf> {
-    if system {
-        Ok(PathBuf::from("/Library/LaunchDaemons"))
+fn plist_dir(system: bool, auto_start: bool, name: &str) -> anyhow::Result<PathBuf> {
+    if auto_start {
+        if system {
+            Ok(PathBuf::from("/Library/LaunchDaemons"))
+        } else {
+            Ok(home_dir()?.join("Library/LaunchAgents"))
+        }
     } else {
-        Ok(home_dir()?.join("Library/LaunchAgents"))
+        unix::storage_dir(name)
```

Comment:
How it is going to work? I don't think launchd recognize units in arbitrary dirs (or does it?). I that's the intention, let's not write it at all?

But the idea is that manually run services you can start via `edgedb server start xxx`. `--start-conf=manual` just means that service should not be started at system startup (or user login).

Reply:
> But the idea is that manually run services you can start via edgedb server start xxx. --start-conf=manual just means that service should not be started at system startup (or user login).

Yes I think we're on the same page.

> How it is going to work? I don't think launchd recognize units in arbitrary dirs (or does it?). I that's the intention, let's not write it at all?

It turns out that launchd will load plist from arbitary dirs if given explicitly, and manages it normally as instructed (e.g. `KeepAlive + SuccessfulExit=false` will restart the service if it's killed with SIGKILL, but not for SIGTERM). However, launchd won't remember plist files from arbitary dirs, that means those manually-loaded services won't be automatically started after reboot or re-login, which is exactly what we need.

Reply:
Okay.